helpers/securewebhook/securewebhook.go

```diff
@@ -0,0 +1,30 @@
+package securewebhook
+
+import (
+	"encoding/json"
+)
+
+// Settings ...
+type Settings struct {
+	Enable *bool `json:"enabled,omitempty"`
+}
+
+// NewSettings ...
+func NewSettings() *Settings {
+	return &Settings{}
+}
+
+// SetEnable ...
+func (s *Settings) SetEnable(enable bool) *Settings {
```

Comment:
Why does this return a `*Settings` if `s` is already a receiver?

Reply:
Followed the pattern from https://github.com/sendgrid/sendgrid-go/blob/master/helpers/mail/mail_v3.go

Reply:
update